src/CMakeLists.txt

```diff
@@ -326,12 +327,16 @@ if(TINYSPLINE_INSTALL_LIBRARY_DIR STREQUAL "")
 	endif()
 endif()
 
+set(TINYSPLINE_INSTALL_BINARY_DIR "bin" CACHE STRING "Installation directory of binary files (relative to CMAKE_INSTALL_PREFIX).")
+
 set(TINYSPLINE_INSTALL_CMAKE_CONFIG_DIR "cmake" CACHE STRING "Installation directory of CMake config files (relative to TINYSPLINE_INSTALL_LIBRARY_DIR).")
 
 set(TINYSPLINE_INSTALL_PKGCONFIG_DIR "pkgconfig" CACHE STRING "Installation directory of pkg-config files (relative to TINYSPLINE_INSTALL_LIBRARY_DIR).")
 
 option(TINYSPLINE_FLOAT_PRECISION "Build TinySpline with float precision." OFF)
 
+option(TINYSPLINE_WARNINGS_AS_ERRORS "Treat warnings as errors" ON)
```

Comment:
All other options were documented above. One may find this superfluous, yet, it allows to add some more internal documentation. In this case you could list the compiler flags enabled with this option.

Reply:
I forgot to add it to the documentation.
It's good to keep it up to date.